config_binance.json.example

```diff
@@ -59,7 +59,7 @@
         "enabled": false,
         "process_throttle_secs": 3600,
         "calculate_since_number_of_days": 7,
-        "total_capital_in_stake_currency": 0.5,
+        "capital_available_percentage": 0.5,
```

Comment:
think you're missing `config.json.example` ...

Reply:
yep !